docs/pages/style-spec.js

```diff
@@ -1115,60 +1115,40 @@ export default class extends React.Component {
+                                <li>Mathematical operators for performing arithmetic and other operations on numeric values</li>
+                                <li>Logical operators for manipulating boolean values and making conditional decisions</li>
+                                <li>String operators for manipulating strings</li>
+                                <li>Data operators, providing access to the properties of source features</li>
+                                <li>Camera operators, providing access to the parameters defining the current map view</li>
+                            </ul>
+
+                            <p>Expressions are represented as JSON arrays. The first element of an expression array is a
+                                string naming the expression operator, e.g. <a href="#expressions-*"><code>"*"</code></a>
+                                or <a href="#expressions-case"><code>"case"</code></a>. Subsequent elements (if any)
+                                are the <em>arguments</em> to the expression. Each argument is either a literal value
+                                (a string, number, boolean, or <code>null</code>), or another expression array.</p>
 
-                            <h3>Syntax</h3>
-                            <p>Mapbox GL JS expressions uses a Lisp-like syntax, represented using JSON arrays. Expressions follow this format:</p>
                             <div className='col12 space-bottom'>
                                 {highlightJSON(`[expression_name, argument_0, argument_1, ...]`)}
                             </div>
-                            <p>The <code>expression_name</code> is the name of a specific expression, e.g. <a
-                                href="#expressions-*"><code>'*'</code></a> or <a
-                                href="#expressions-case"><code>'case'</code></a>. Each <code>argument_i</code> is either
-                                a literal value (a string, number, boolean, or <code>null</code>), or else another
-                                expression in the array form above.</p>
-
-                            <h3>Property expressions</h3>
-                            <p>A <a id="property-expression" className="anchor"></a><strong>property expression</strong> is
-                                any expression defined using an expression that includes a reference to feature property
-                                data. Property expressions allow the appearance of a feature to change with its
-                                properties. They can be used to visually differentate types of features within the same
-                                layer or create data visualizations.</p>
-
-                            <h4>Example: a property expression</h4>
+
+                            <h3>Data expressions</h3>
+                            <p>A <em>data expression</em> is any expression that uses one of the data operators:
```

Comment:
For some readers, it may be a bit easier to grasp this if we couple the list-based definition with a conceptual definition.  Something like:

```
A <em>data expression</em> is any expression that access feature data -- that is, any expression that uses one of the data operators:
```